### PR TITLE
add pagination on rawsql results page (close #5002)

### LINKED ISSUE #5002: console freezes when a query with lots of rows as response is executed from raw sql page
possible solution would be to limit the number of results that are displayed at a time on the and allow pagination if possible

### PR DESCRIPTION
**Pull request by soorajshankar**
### Description
this PR will add pagination feature on rawSQL results page which makes console more performant, by doing this large data will not be displayed in a single big table, instead the pagination take care of the data rendering.

### Changelog

- [x] `CHANGELOG.md` is updated with user-facing content relevant to this PR. If no changelog is required, then add the `no-changelog-required` label.

### Affected components

- [ ] Console

### Related Issues
close #5002  

### Solution and Design
added pagination using react table 

### Steps to test and verify
Run a SQL query that has huge number of results and verify the pagination feature.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -31,6 +31,7 @@ If you do have such headers configured, then you must update the header configur
 - server: add logs for action handlers
 - server: add request/response sizes in event triggers (and scheduled trigger) logs (#5463)
 - server: change startup log kind `db_migrate` to `catalog_migrate` (#5531)
+- console: add pagination on rawsql results page (close #5002) (#5629)
```
Comment (tirumaraiselvan) [inline CHANGELOG.md:34]: Can you take this to `Next release` section pls.

Comment (soorajshankar) [reply CHANGELOG.md]: oops, sorry changing now... :)

File: console/src/components/Services/Data/RawSQL/ResultTable.tsx [hunk 2]
```diff
@@ -0,0 +1,42 @@
+import React, { useMemo } from 'react';
+import ReactTable from 'react-table';
+import styles from '../../../Common/TableCommon/Table.scss';
+
+interface ResultTableProps {
+  headers: string[];
+  rows: Array<string[]>;
```
Comment (iykekings) [inline console/src/components/Services/Data/RawSQL/ResultTable.tsx:7]: Looks a lot simpler
```suggestion
  rows: string[][];
```

File: console/src/components/Services/Data/RawSQL/ResultTable.tsx [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+}
+const getTableConfig = (headers: string[], rows: Array<string[]>) => {
+  const columns = headers.map(i => ({
+    Header: i,
+    accessor: i,
+  }));
+  const data = rows.map(i => {
+    const result: Record<any, string> = {};
```
Comment (iykekings) [inline console/src/components/Services/Data/RawSQL/ResultTable.tsx:15]: Would this work instead?
```suggestion
    const result: Record<string, string> = {};
```

**Review by iykekings — COMMENTED** [review]
Just few suggestions, everything looks perfect. 👌

File: console/src/components/Services/Data/RawSQL/ResultTable.tsx (hunk 3 above)
Comment (soorajshankar) [reply console/src/components/Services/Data/RawSQL/ResultTable.tsx:15]: done

**Pull request closed without merging**